packages/fa-icon-chooser/dev/runtime.js

```diff
@@ -72,7 +72,11 @@ const FaIconChooserDevExports = (function () {
     document.querySelectorAll('#result pre').forEach(child => child.remove());
   }
 
+  const svgEmbedOverrideCallback = (window['__FA_SVG_EMBED__'] = () => true);
```

Comment:
We don't need to invoke the callback here in `runtime.js`. We just need to set up the global so that the `fa-icon-chooser` can invoke it if it's present.

```suggestion
  window['__FA_SVG_EMBED__'] = () => true;
```